core/src/main/resources/hudson/views/BuildButtonColumn/column.properties

```
@@ -23,3 +23,4 @@
 Task_scheduled={0} scheduled
 Schedule_a_task=Schedule a {1} for {0}
 Schedule_a_task_with_parameters=Schedule a {1} with parameters for {0}
+Task_schedule_failed=Failed to schedule build for {0}.
```

```suggestion
Task_schedule_failed=Failed to schedule build for {0}
```

For consistency with the 'Build scheduled' content